### PR TITLE
Fixing ci failures

### PR DESCRIPTION
**Pull request by adamdubiel**

### CODE REVIEW

File: hermes-test-helper/src/main/java/pl/allegro/tech/hermes/test/helper/environment/KafkaStarter.java [hunk 1]
```diff
@@ -2,7 +2,9 @@
 
 import com.jayway.awaitility.Duration;
 import kafka.server.KafkaConfig;
+import kafka.server.KafkaServer;
```
Comment (chemicL) [inline hermes-test-helper/src/main/java/pl/allegro/tech/hermes/test/helper/environment/KafkaStarter.java]: Unused imports added

**Comment by adamdubiel** [comment]
do not review, pr created to trigger builds

File: integration/src/integration/java/pl/allegro/tech/hermes/integration/KafkaSingleMessageReaderTest.java [hunk 2]
```diff
@@ -68,8 +68,8 @@ public void shouldFetchSingleAvroMessage() throws IOException {
                 .withContentType(AVRO).build();
         operations.buildTopic(topic);
         wait.untilTopicDetailsAreCreated(topic.getName());
-        assertThat(publisher.publish(topic.getQualifiedName(), avroUser.asBytes())
-                .getStatus()).isEqualTo(CREATED.getStatusCode());
+
+        publisher.publish(topic.getQualifiedName(), avroUser.asBytes(), Response.Status.CREATED);
```
Comment (chemicL) [inline integration/src/integration/java/pl/allegro/tech/hermes/integration/KafkaSingleMessageReaderTest.java]: I'd argue that
`import static pl.allegro.tech.hermes.integration.test.HermesAssertions.assertThat

assertThat(publisher.publish(topic.getQualifiedName(), avroUser.asBytes()).hasStatus(CREATED);`

would be more verbose and clear at this stage than having it tested inside the publisher.

File: integration/src/integration/java/pl/allegro/tech/hermes/integration/PublishingAvroTest.java [hunk 3]
```diff
@@ -57,10 +58,9 @@ public void shouldPublishAvroAndConsumeJsonMessage() throws InterruptedException
         operations.createSubscription(topic, "subscription", HTTP_ENDPOINT_URL);
 
         // when
-        Response response = publisher.publish("publishAvroConsumeJson.topic", user.asBytes());
```
Comment (chemicL) [inline integration/src/integration/java/pl/allegro/tech/hermes/integration/PublishingAvroTest.java]: Same remarks as in KafkaSingleMessageReaderTest

**Comment by druminski** [comment]
:+1:

**Comment by chemicL** [comment]
:+1:

**Pull request merged**